docs/guides/tools-reference/file-transfer/comparing-data-transfer-utilities/index.md

```diff
@@ -0,0 +1,128 @@
+
+### Data Storage Size and Type
+
+Along with how much data is being transferred, the kind of data and the destination are equally important (e.g., your data’s source and target destination, their respective file systems, and how much storage is required once the data is at rest).
+
+For example, most end users only deal with traditionally structured file systems when using their local environment, with data organized hierarchically into folders. In contrast, block storage breaks up files into separate blocks and stores them separately, allowing for significant performance benefits. Object storage saves files in a non-hierarchical, self-contained “flat” format. This allows for increased scalability and sizable reductions in cost and complexity.
+
+### File Type and Size
+
+Different file types and sizes also call for different data transfer utilities. For example, server log files grow in size quickly but can be easily compressed, whereas media files (e.g., MP4 video files, WAV sounds files) are usually quite large, aren’t easily compressed, and typically do not change once saved. Your choice of data transfer utility will depend on what kind of files are being transferred.
+
+Different file types also involve varying metadata considerations. Object storage is self-contained, so it includes both data and metadata for easy search and access and retrieval. Structured file system storage also includes extensive metadata; when right-clicking a file in your desktop operating system, you can view its create/modify time, file type, version number, and more. Alternatively, block storage is highly performant but provides no metadata capabilities.
+
+### One-time Transfer vs. Recurring Syncs
+
+A specific utility may be well-suited for your use case depending on whether the data transfer is a single event or a recurring job. For example, an ad-hoc file transfer to a cloud server may only require a desktop-based data transfer utility, while a regularly scheduled backup job may require an automated tool or process that works at scale.
+
+## Comparing Data Transfer Utilities
+
+### Rsync
+
+[Rsync](https://rsync.samba.org/) is a standard Linux-based data transfer utility designed for syncing files between remote and local servers. Known for its flexibility, speed, and scriptable configuration, rsync is often a go-to utility for Linux administrators looking to quickly transfer and synchronize files between systems.
+
+#### Use Cases
+
+Rsync is ideal for syncing files between local and remote Linux machines in incremental backup and transfer use cases, as well as customizing large and complicated sync jobs with specific options and settings, like selecting a particular Linux shell to use in the transfer, or specifying files to analyze and exclude. Its delta-transfer algorithm reduces network traffic during syncs by only sending parts of a file that differ from the files on the recipient machine.
```

Comment:
How do Linux shells interact with how rsync is invoked? I'm not familiar with that